### ISSUE TITLE
Events Alarming/Notification (API?)

### ISSUE THREAD

**Issue by 4Ykw** [aaroncox/vessel#26]
To further complete/increase the functionality of Vessel, I was thinking:

Can Vessel (while running) send an email with some kind of resume/events of selectable options like...
- New claimable balances available
- Vested SP (when converted)
- Account STEEM or SBD changes
- Others

Note: the ideal would be to notify via API other tools (like eSteem or steemit.chat, etc), since email is somehow centralised. Joint collaboration would be a must here, so maybe this (API) would be something to hook into steem itself first. What does your gut tell you?

**Comment by aaroncox** [comment]
I don't think the app itself would be able to do this without a significant security risk (to the API issuing the notifications, not to the wallet). Any service that does things like this would require an API key, which would have to be baked into Vessel, which could then be extracted.

A completely independent 3rd party solution would probably be the best choice for a service like this, which then could be toggled on/off via Vessel perhaps.

**Comment by 4Ykw** [comment]
Yes. Thought so too. But I was trying to confirm. Anyway, maybe for later stages (not that is an highly desired feature from global population I would guess).
